config/pop-os/22.04.mk

```diff
@@ -52,6 +52,9 @@ LIVE_PKGS=\
 	pop-installer-casper \
 	pop-shop-casper
 
+# Staging Branches to have in the live instance (these package must alse be installed)
```

Comment:
typo: these packages must also be installed

Reply:
a420bff855d8a6256c1b4cc34262b6af2f5ba875